hypervisor/dm/vpci/vpci.c

```diff
@@ -132,7 +112,18 @@ void vpci_init(struct acrn_vm *vm)
 #endif
 
 	if ((vpci->ops->init != NULL) && (vpci->ops->init(vm) == 0)) {
-		register_io_emulation_handler(vm, PCI_PIO_IDX, &pci_cfg_range, pci_cfg_io_read, pci_cfg_io_write);
+		pci_cfg_range.flags = IO_ATTR_RW;
+		pci_cfg_range.base = (uint16_t)PCI_CONFIG_ADDR;
+		pci_cfg_range.len = (uint16_t)4U;
```

Comment:
please use the original style. use static array rather than code

Reply:
OK